include/measurement_kit/ooni/ooni_reporter.hpp

```diff
@@ -13,8 +13,8 @@ class OoniReporter : public BaseReporter {
   public:
     static Var<BaseReporter> make(const OoniTest &ooni_test);
 
-    Continuation<Error> open(const Report &report) override;
-    Continuation<Error> write_entry(const Entry &entry) override;
+    Continuation<Error> open(Report report) override;
+    Continuation<Error> write_entry(Entry entry) override;
```

Comment:
Since now we have a chain of continuations, better to copy report and entry because there are many lambdas and it is not wise to pass references and then have to remember to make copies to keep safe objects. Better to change the API and take the safe side in one single place and for good.